svgeditor/scripts.js

```diff
@@ -0,0 +1,97 @@
+// Order of operations
+// 1. Create UI
+// 2. Draw first SVG
+// 3. Event listener actions:
+// 3a. Create event listener
+// 3b. Update SVG
+// 3c. Redraw SVG
+
+const canvasId = 'canvas0',
+  defaultSvg = '/assets/sampleSVG.svg',
+  inputId = 'text-entry-box',
+  objId = 'svg-object',
+  textId = 'textID';
+
+const init = () => {
```

Comment:
ah, I always forget about arrow functions and they throw my eye off. So from what I can see you've essentially got two groups of function "init" and "updateDrawing". Init runs on window load and then at the end of the init function it triggers update drawing. I havent see someone wrap groups of functions like this but from what i can tell the notation is just 
const variable = { 
    fName: function(){
        //do stuff
    }}
Then you call it with dot notation like "variable.fName()". I really like that. It takes me a second to reorganize in my head but it's so much cleaner.